packages/app/src/app/pages/Sandbox/Editor/Workspace/Project/Project.tsx

```diff
@@ -36,159 +36,149 @@ import { Keywords } from './Keywords';
 import { SandboxConfig } from './SandboxConfig';
 import { Title } from './Title';
 
-interface IProjectProps {
+type Props = {
   editable?: boolean;
-  store: any;
-  signals: any;
-}
-
-export const Project = inject('store', 'signals')(
-  hooksObserver(
-    ({
-      editable,
-      store: { editor, isPatron },
-      signals: {
-        workspace: { sandboxPrivacyChanged },
-      },
-    }: IProjectProps) => {
-      const sandbox = editor.currentSandbox;
-      const template = getTemplateDefinition(sandbox.template);
-      const { isServer } = template;
-
-      return (
-        <Container>
-          <BasicInfo>
-            <Title editable={editable} />
-            <Description editable={editable} />
-            {/* Disable until we also moved SSE over */}
-            {/* <Alias editable={editable} /> */}
-          </BasicInfo>
-
-          {!sandbox.team && sandbox.author && (
-            <Author author={sandbox.author} />
-          )}
-
-          {sandbox.team && (
+};
+export const Project: FunctionComponent<Props> = ({ editable }) => {
+  const {
+    actions: {
+      workspace: { sandboxPrivacyChanged },
+    },
+    state: {
+      editor: { currentSandbox: sandbox },
+      isPatron,
+    },
+  } = useOvermind();
+
+  const template = getTemplateDefinition(sandbox.template);
+
+  return (
+    <Container>
+      <BasicInfo>
+        <Title editable={editable} />
+
+        <Description editable={editable} />
+
+        {/* Disable until we also moved SSE over */}
+        {/* <Alias editable={editable} /> */}
+      </BasicInfo>
+
+      {!sandbox.team && sandbox.author && <Author author={sandbox.author} />}
+
+      {sandbox.team && (
+        <Tooltip appendTo="parent" content="This sandbox is owned by this team">
+          <Item style={{ color: 'white', display: 'flex' }}>
+            <TeamIcon style={{ fontSize: '1.125em', marginRight: '.5rem' }} />
+
+            <div>{sandbox.team.name}</div>
+          </Item>
+        </Tooltip>
+      )}
+
+      {sandbox.git && (
+        <Item>
+          <GithubBadge
+            url={githubRepoUrl(sandbox.git)}
+            username={sandbox.git.username}
+            repo={sandbox.git.repo}
+            branch={sandbox.git.branch}
+          />
+        </Item>
+      )}
+
+      <StatsContainer>
+        <Stats />
```

Comment:
```suggestion
        <Stats sandbox={sandbox} />
```

This one needs the sandbox :)